internal/cluster/phase.go

```diff
@@ -41,17 +42,17 @@ type OperationPhaseOption struct {
 	Value any
 }
 
-func NewOperationPhase(folder string) (*OperationPhase, error) {
+func NewOperationPhase(folder string, kfdTools config.KFDTools) (*OperationPhase, error) {
 	basePath := folder
 
-	vendorPath, err := filepath.Abs(path.Join(basePath, "../", "vendor"))
+	binPath, err := filepath.Abs(path.Join(basePath, "../../", "bin"))
```

Comment:
I am not sure this works every time, considering we are passing bin path as a flag in some commands